person_capture/gui_app.py

```
@@ -1749,6 +1756,430 @@ def _shift_crop_to_include_box(
+            x1 = px1
+        if px2 > x1 + crop_w:
+            x1 = px2 - crop_w
+        if py1 < y1:
+            y1 = py1
+        if py2 > y1 + crop_h:
+            y1 = py2 - crop_h
+        x1 = max(bx1, min(bx2 - crop_w, x1))
+        y1 = max(by1, min(by2 - crop_h, y1))
+        x2 = x1 + crop_w
+        y2 = y1 + crop_h
+        return self._clip_to_frame(x1, y1, x2, y2, int(round(bx2)), int(round(by2)))
```

Ruff (0.15.11)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)